spec/features/bulk_screen_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+# Feature/view tests for the (old) bulk actions view.
+feature 'Bulk actions view', js: true do
+  before :each do
+    @current_user = double(
+      :webauth_user,
+      login: 'sunetid',
+      logged_in?: true,
+      permitted_apos: [],
+      is_admin: true,
+      permitted_collections: [ [ "string" ] ]
```

Comment:
Space inside square brackets detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.